### PR TITLE
doc: add meeting notes 2023-02-20 (#58)

### LINKED ISSUE #58: Node.js  Performance Team Meeting 2023-02-20
## Time

**UTC Mon 20-Feb-2023 18:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Mon 20-Feb-2023 10:00 (10:00 AM) |
| US / Mountain | Mon 20-Feb-2023 11:00 (11:00 AM) |
| US / Central  | Mon 20-Feb-2023 12:00 (12:00 PM) |
| US / Eastern  | Mon 20-Feb-2023 13:00 (01:00 PM) |
| EU / Western  | Mon 20-Feb-2023 18:00 (06:00 PM) |
| EU / Central  | Mon 20-Feb-2023 19:00 (07:00 PM) |
| EU / Eastern  | Mon 20-Feb-2023 20:00 (08:00 PM) |
| Moscow        | Mon 20-Feb-2023 21:00 (09:00 PM) |
| Chennai       | Mon 20-Feb-2023 23:30 (11:30 PM) |
| Hangzhou      | Tue 21-Feb-2023 02:00 (02:00 AM) |
| Tokyo         | Tue 21-Feb-2023 03:00 (03:00 AM) |
| Sydney        | Tue 21-Feb-2023 05:00 (05:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Performance%20Team+Meeting+2023-02-20&iso=20230220T1800
* or https://www.wolframalpha.com/input/?i=06PM+UTC%2C+Feb+20%2C+2023+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1nBxnL0iElhiu-7vAlRWNWpqLHBv_lacr3RXFEdBZG70/edit>

## Agenda

Extracted from **performance-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/performance

* getLibuvNow is slow [#49](https://github.com/nodejs/performance/issues/49)
* AbortSignal pattern is slow [#44](https://github.com/nodejs/performance/issues/44)
* Module resolution [#39](https://github.com/nodejs/performance/issues/39)
* WHATWG Streams [#9](https://github.com/nodejs/performance/issues/9)

## Invited

* Performance team: @nodejs/performance

## Notes

The agenda comes from issues labelled with `performance-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/94710602723

### PR DESCRIPTION
**Pull request by UlisesGascon**
Close #58

### CODE REVIEW

File: meetings/2023-02-20.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* **GitHub Issue**: https://github.com/nodejs/performance/issues/58
+
+## Present
+
+* Performance team: @nodejs/performance
+* Yagiz Nizipli: @anonrig
+* Ulises Gascon: @ulisesGascon
+* Santiago Gimeno: @santigimeno
+* Juan Arboleda: @soyjuanarbol
+
```
Comment (anonrig) [inline meetings/2023-02-20.md:15]: @ronag and @BridgeAR are missing from the list (afaik)

File: meetings/2023-02-20.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Agenda
+
+## Announcements
+
+- The new URL parser will be released this week!
```
Comment (tniessen) [inline meetings/2023-02-20.md:21]:
```suggestion
- The new URL parser will be released this week!
```

File: meetings/2023-02-20.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- It was a productive week for all of us! Other improvements had been made by the group members (Ruben, Joyee and multiple other collaborators) based on separated issues
+- Santiago is planning to do a new PR for event creation this week
+
+*Extracted from **performance-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/performance
+
+* getLibuvNow is slow [#49](https://github.com/nodejs/performance/issues/49)
+  - Seems that there is a fix for it, related to timers (https://github.com/nodejs/node/pull/46579) with v8 fastApi. (Thanks to joyee!)
+It will be removed from the agenda
```
Comment (tniessen) [inline meetings/2023-02-20.md:31]:
```suggestion
  It will be removed from the agenda
```

File: meetings/2023-02-20.md (hunk 1 above)
Comment (UlisesGascon) [reply meetings/2023-02-20.md]: Solved in [88fa5d7](https://github.com/nodejs/performance/pull/61/commits/88fa5d717fa011508176b4bbd3bc8690d20fa1c0)

**Comment by metcoder95** [comment]
I just wanted to mention that the PR for #38 is already open and getting feedback (ref: https://github.com/nodejs/node/pull/46607). I'm planning to address feedback and adjust tests during this week 🙂 

Sadly I cannot attend the meetings as it overlaps with other personal events at the time, but I'm keeping in sync through these docs, and I'll try to share updates of any in this form. Thanks! 🙂

**Review by anonrig — APPROVED** [review]

**Pull request merged**